cfbs/module.py

```diff
@@ -3,6 +3,10 @@
 from cfbs.pretty import pretty
 
 
+def is_module_added_manually(added_by: str):
+    return (added_by == "cfbs add") or (added_by == "cfbs init")
```

Comment:
You could consider
```suggestion
    return added_by in ("cfbs add", "cfbs init")
```
but there is nothing wrong with what you did ;)